sycl/test/regression/same_unnamed_kernels.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: %clangxx -fsycl %s -o %t.out -fsycl-unnamed-lambda
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+
```

Comment:
Why aren't other devices necessary? Merely asking, as I'm probably missing the context.

Reply:
This test is device-agnostic. We get fail in frontend, so it doesn't matter which backend executes it.

Reply:
Please, add Clang regression test.